map/framework.cpp

```diff
@@ -2855,9 +2854,11 @@ void SetHostingBuildingAddress(FeatureID const & hostingBuildingFid, DataSource
 }
 }  // namespace
 
-bool Framework::CanEditMap() const
+bool Framework::CanEditMapForPosition(m2::PointD const & position) const
 {
-  return !GetStorage().IsDownloadInProgress();
+  auto const & storage = GetStorage();
+  ASSERT(m_infoGetter, ("CountryInfoGetter shouldn't be nullprt."));
+  return IsPointCoveredByDownloadedMaps(position, storage, *m_infoGetter) && storage.IsAllowedToEditVersion(m_infoGetter->GetRegionCountryId(position));
```

Comment:
I suspect that it should work without first IsPointCoveredByDownloadedMaps:
GetRegionCountryId -> returns empty -> status returns NotDownloaded.

Reply:
Fixed and tested - it works as expected without the `IsPointCoveredByDownloadedMaps`. Thanks!